app/views/atmosphere/admin/funds/_user_fund.html.haml

```diff
@@ -0,0 +1,36 @@
+.btn-group
+  - user = user_fund.user
+
+  %span.btn.btn-default.btn-xs.disabled.non-opaque
+    = user.full_name
+    = surround '(', ')' do
+      = user.login
+
+  - if user_fund.default
+    %span{ class: 'btn btn-xs btn-primary has_bottom_tooltip',
+    title: t('funds.set_default.to_false',
+             user: user_fund.user.full_name) }
+      D
+  - else
+    .btn-group
+      = form_for [:admin, user_fund] do |f|
```

Comment:
This have one small drawback - whole webpage is refreshed and selection for fund graph disappeared. This is very, very minor issue and if complicated don't have to be changed :smile: